src/js/grid.js

```diff
@@ -49,7 +54,9 @@ Grid.prototype = {
     },
 
     _html: function () {
-        var html = '<div class="medium-editor-table-builder-grid clearfix">';
+        var width = this.columns * 16 + 2,
```

Comment:
You're right. I'll make it a constant. Width of the grid was previously hardcoded to 162px in css - 16px is width of one cell and 2px is width of border = 162px is width of the grid with 10 columns. Now the width is calculated dynamically depending on a number of columns.